app/redux/reducers/SetsReducer.js

```diff
@@ -162,6 +168,12 @@ const saveWorkoutSet = (state, action) => {
     });
 };
 
+// DELETE_WORKOUT_SET
+
+const deleteWorkoutSet = (state, action) => {
+    return state.workoutData.filter(set => set.setID !== action.setID);
+};
```

Comment:
- [ ] this isn't returning the state properly, it's returning just the filtered workoutdata. Need to return the entire sets state